### PR TITLE
put in a option for fire fox

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR shows a preview of what `datatable` header would look like in FireFox when you scroll
#### Where should the reviewer start?
aries theme 
This would be taken out and put into the theme 
#### What testing has been done on this PR?
visual 
#### How should this be manually tested?
http://localhost:3000/components/table
go to the link above in firefox
#### Any background context you want to provide?
https://github.com/grommet/grommet-theme-hpe/issues/200
#### What are the relevant issues?
issue 200
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: aries-site/src/themes/aries.js [hunk 1]
```diff
@@ -3,6 +3,40 @@ import { deepMerge } from 'grommet/utils';
...
+  dataTable: {
+    pinned: {
+      header: {
+        extend: ({ theme }) => `
+        backdrop-filter: blur(8px);
+        @-moz-document url-prefix() {
+        background: ${
+          theme.dark ? 'rgba(64, 75, 92, 0.95)' : 'rgba(255, 255, 255, 0.95)'
+        };
+        }
```
Comment (halocline) [inline aries-site/src/themes/aries.js:15]: Can you add a comment for why this is needed and also provide a link to MDN docs / can-i-use docs for support of `backdrop-filter`? Sounds like FireFox should be supporting shortly - currently an opt in users can enable, but not yet a default.

File: aries-site/src/themes/aries.js [hunk 2]
```diff
@@ -3,6 +3,40 @@ import { deepMerge } from 'grommet/utils';
...
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
+  dataTable: {
+    pinned: {
+      header: {
+        extend: ({ theme }) => `
+        backdrop-filter: blur(8px);
+        @-moz-document url-prefix() {
+        background: ${
+          theme.dark ? 'rgba(64, 75, 92, 0.95)' : 'rgba(255, 255, 255, 0.95)'
```
Comment (halocline) [inline aries-site/src/themes/aries.js:13]: Can we translate these to the hexadecimal equivalents for consistency?

Comment (halocline) [reply aries-site/src/themes/aries.js:13]: How were these values arrived at? Did you have a chance to run these by @vavalos5 for a quick 👍 👎 ?

Comment (britt6612) [reply aries-site/src/themes/aries.js:13]: I will change these and no I will run them by her I tried to use the `background-contrast` but it wasn't enough `opactity` still looked pretty bad

File: aries-site/src/themes/aries.js (hunk 1 above)
Comment (britt6612) [reply aries-site/src/themes/aries.js:15]: yes I read that Firefox had `backdrop-filter` on its list of priority

File: aries-site/src/themes/aries.js (hunk 2 above)
Comment (britt6612) [reply aries-site/src/themes/aries.js:13]: @vavalos5 and I spoke and she is okay with these changes she just mentioned to make sure we stay on top of it when it is available for firefox so we can take this out!

File: aries-site/src/themes/aries.js (hunk 1 above)
Comment (britt6612) [reply aries-site/src/themes/aries.js:15]: I will change these values on the PR for the theme and add the comment there then we can strip this out

**Comment by taysea** [comment]
Closing this PR since the theme PR has been merged.

**Pull request closed without merging**